src/admin-views/events-meta-box/date-time.php

```diff
@@ -0,0 +1,111 @@
+<table cellspacing="0" cellpadding="0" id="EventInfo">
+	<tr>
+		<td colspan="2" class="tribe_sectionheader">
+			<div class="tribe_sectionheader" style="">
+				<h4><?php esc_html_e( 'Time &amp; Date', 'the-events-calendar' ); ?></h4></div>
+		</td>
+	</tr>
+	<tr>
+		<td colspan="2">
+			<table class="eventtable">
+				<tr id="recurrence-changed-row">
+					<td colspan='2'><?php esc_html( sprintf( __( 'You have changed the recurrence rules of this %1$s.  Saving the %1$s will update all future %2$s.  If you did not mean to change all %2$s, then please refresh the page.', 'the-events-calendar' ), $events_label_singular_lowercase, $events_label_plural_lowercase ) ); ?></td>
+				</tr>
+				<tr>
+					<td class="tribe-datetime-label"><?php esc_html_e( 'Start/End:', 'the-events-calendar' ); ?></td>
+					<td class="tribe-datetime-block">
+						<input
+							autocomplete="off"
+							tabindex="<?php tribe_events_tab_index(); ?>"
+							type="text"
+							class="tribe-datepicker tribe-field-start_date"
+							name="EventStartDate"
+							id="EventStartDate"
+							value="<?php echo esc_attr( $EventStartDate ) ?>"
+						/>
+						<span class="helper-text hide-if-js"><?php esc_html_e( 'YYYY-MM-DD', 'the-events-calendar' ) ?></span>
+
+						<input
+							autocomplete="off"
+							tabindex="<?php tribe_events_tab_index(); ?>"
+							type="text"
+							class="tribe-timepicker tribe-field-start_time"
+							name="EventStartTime"
+							id="EventStartTime"
+						<?php echo Tribe__View_Helpers::is_24hr_format() ? 'data-format="H:i"' : '' ?>"
```

Comment:
Actually, isn't there also a stray `"` at the end of the line, besides the indent being out?